modules/client/shared/src/main/scala/lucuma/itc/client/IntegrationModeInput.scala

```diff
@@ -29,10 +29,10 @@ final case class SpectroscopyIntegrationTimeInput(
   mode:            InstrumentMode
 )
```

Comment:
So this works for either imaging or spectroscopy?  Should we rename the file `IntegrationTimeInput.scala`?

Reply:
Right you can pass any mode. I'll rename the file

Reply:
I'm a little confused because the API has different input classes for imaging and spectroscopy.  The `Encoder` here always seems to make the spectroscopy one?